### PR TITLE
Fixes DOC-1400, moderation prigs not part of course team privs

### PR DESCRIPTION
**Pull request by lamagnifica**
@mlkwaqas, I added these changes in addition to previously added notes
in these sections of the Building & Running an edX Course guide:
http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/r
unning_course/discussions.html#assigning-discussion-roles
http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/r
unning_course/course_staffing.html#staffing
http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/b
uilding_course/creating_new_course.html#add-course-team-members
@srpearce please review

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/course_staffing.rst [hunk 1]
```diff
@@ -60,6 +60,10 @@ as the course staff. They can also:
 * Add and remove Discussion Admins, Discussion Moderators, and Discussion
   Community TAs.
 
+.. note:: For a team member to be able to moderate the course discussions,
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/running_course/course_staffing.rst]: I wonder if there's a smoother way to say this - "A team member cannot moderate course discussions until you separately assign one of the discussion roles to her"?

**Comment by srpearce** [comment]
One comment, otherwise :+1:

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 2]
```diff
@@ -158,9 +158,11 @@ To add a course team member:
 #. Click **Add a New Team Member**.
 #. Enter the new team member's email address, then click **ADD USER**. 
 
-The new team member can now work on the course in Studio. To preview
-the course in the LMS or work on the Instructor Dashboard, the team member
-must enroll in the course.
+The new team member can now work on the course in Studio. To preview the
+course in the LMS or work on the Instructor Dashboard, the team member must
```
Comment (mlkwaqas) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: To work on Instructor Dashboard you don't need to be enrolled in the course. You can use Instructor Dashboard if you are global staff or someone adds you in course staff but in both way you won't be able to see Lms side of course. So I guess you should exclude `or work on the Instructor Dashboard` from here

**Comment by mlkwaqas** [comment]
Just one comment, otherwise lgtm :+1:

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: Ah, I see the distinction now, thank you.

File: en_us/shared/building_and_running_chapters/running_course/course_staffing.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_staffing.rst]: I prefer a positive construction to a negative one... thoughts?

**Pull request merged**